crates/sui-swarm-config/src/genesis_config.rs

```diff
@@ -201,6 +201,17 @@ impl GenesisConfig {
             });
         }
 
+        // Build an extra faucet address with allocation for sui-test-validator
+        let (address, keypair) = get_key_pair_from_rng(&mut rng);
```

Comment:
I still don't understand why we need this here. Can we not add a new account outside wherever we are trying to create a genesis config?